applications/daily_tests/automatic_sync_test.js

```diff
@@ -9,38 +9,72 @@ const SyncType = {
 };
 
 async function main() {
-  let syncType;
+  let syncType, forced;
   if (process.argv.includes("pruned")) {
     syncType = SyncType.Pruned;
   } else {
     syncType = SyncType.Archival;
   }
 
-  const { blockHeight, timeDiffMinutes, blockRate } = await run({
-    log: "./logs/base-node.log",
-    syncType,
-  });
+  // Check for forced index.
+  let index = process.argv.findIndex((element) => element === "forced");
+  if (index >= 0) {
+    forced = parseInt(process.argv[index + 1]);
+    if (isNaN(forced)) {
+      // Check if index is specified.
+      console.log("Wrong forced parameter");
+      return;
+    }
+  }
+  try {
+    const { blockHeight, timeDiffMinutes, blockRate, forcedSyncPeer } =
+      await run({
+        log: "./logs/base-node.log",
+        syncType,
+        forced,
+      });
 
-  console.log(
-    `${syncType} sync to block height ${blockHeight} took ${timeDiffMinutes} minutes for a rate of ${blockRate} blocks/min`
-  );
+    console.log(
+      `${syncType} sync ${
+        forcedSyncPeer ? "forced to " + forcedSyncPeer : ""
+      } to block height ${blockHeight} took ${timeDiffMinutes} minutes for a rate of ${blockRate} blocks/min`
+    );
+  } catch (err) {
+    console.log(err);
+  }
 }
 
 async function run(options) {
+  let forcedSyncPeer;
   const baseNode = new BaseNodeProcess("compile", true);
   await baseNode.init();
 
+  let config = await fs.readFile(baseNode.baseDir + "/config/config.toml");
+
   // Set pruning horizon in config file if `pruned` command line arg is present
   if (options.syncType === SyncType.Pruned) {
-    let config = await fs.readFile(baseNode.baseDir + "/config/config.toml");
-    let updated_config = config
+    config = config
       .toString()
       .replace("#pruning_horizon = 0", "pruning_horizon = 1000");
-    await fs.writeFile(
-      baseNode.baseDir + "/config/config.toml",
-      updated_config
+  }
+
+  // If we have forced index, rewrite the config.
+  if (options.forced !== undefined) {
```

Comment:
Can an ENV not be used here instead?

Reply:
e.g. TARI_WEATHER_WAX__FORCE_SYNC_PEERS

Reply:
There is no support for force sync env yet. But there will be after my second PR.
So I'm going to change the it to env.

Reply:
Done